src/tools/cephfs/cephfs-shell

```diff
@@ -57,6 +58,29 @@ except ImportError:
 
 cephfs = None   # holds CephFS Python bindings
 shell = None    # holds instance of class CephFSShell
+exit_codes = {'Misc': 1,
```

Comment:
Won't it be better to align our exit values with the standard one? Setting `EPERM` to 1 `ENOENT` to 2 and so on.

Reply:
imo it is fine as we are defining custom exit values.

Reply:
I don't mind creating custom values, IMO we must align them as far as possible.